.circleci/config.yml

```diff
@@ -231,6 +231,16 @@ jobs:
       - run: date +%s > "pnpm-lock.yaml"
       - *cache_save
 
+  #
+  # Check for dead local links
+  #
+  dead_links_check:
+    docker:
+      - image: 'lycheeverse/lychee:0.14.3'
```

Comment:
Pinned, we could also use the latest but then we risk surprise failures to pass on PRs that don't touch links.